integration-tests/tests/src/tests/sync/open-behavior.ts

```diff
@@ -126,7 +129,6 @@ describe("OpenBehaviour", function () {
       sync: {
         user,
         partitionValue,
-        //@ts-expect-error const enum is removed at runtime, should either remove const or leave this as is.
```

Comment:
I actually don't get an error even after restarting the TS server? I'm not sure if something changed since this comment was added, as I'm not sure what would be the error 😁

Reply:
Interesting, I do see the errors if not using them. Since `_sessionStopPolicy` is internal, it's stripped from the types so TS would say that it doesn't exist on the `sync` config. So we usually use the `@ts-expect-error` for e.g. internal fields/methods.
```ts
export type BaseSyncConfiguration = {
  // ...

  /** @internal */
  _sessionStopPolicy?: SessionStopPolicy;
};
```
The other thing is that the type provided for `_sessionStopPolicy` is also the incorrect TS type (`string` is provided rather than `SessionStopPolicy` enum value).

Reply:
Got it! I've reverted my change, I'm not really sure why it doesn't raise an error for me, but I would probably guess I have something setup in the wrong way 😁

Reply:
It does behave a bit strange at times 😉